### DISCUSSION TITLE
有关时间线和记录的建议

### DISCUSSION THREAD

**Discussion by eamooooon** [shion-app/shion#48] [Ideas]
①有时候想查看当天的前几个小时的使用记录 但是每次点到时间线都要从早上一直往下翻 所以是否可以加入反向的按钮 或者点进时间线自动到页底
②记录这块可以加入一键使用上次计划/标签的功能 这样比较便捷 同时也可以加入快捷键 更方便快速的开始记录

**Comment by hanaTsuk1** [comment]
1. `时间线自动到底`加入开发计划
2. 加入`上一次使用`和`快捷键`不太考虑，标签页面可以选择`开始计时`，不知道你尝试过没有

**Reply by eamooooon** [reply]
谢谢 这我还真没发现 用的时间还不是很多
